### PR TITLE
Feature/ivyportal 17297 implement feature hide fullscreen mode widget infor icons in the other widgets in lts

### PR DESCRIPTION
**Pull request by lmluat-axonivy**
- implement feature hide fullscreen mode widget infor icons in the other widgets in lts
+ Process widget
+ Process viewer widget
+ Custom widget
+ News feed widget

- Update gui test & documentation

### CODE REVIEW

File: Documentation/portal-guide/source/portal-developer-guide/dashboard/new-dashboard-process-widget.rst [hunk 1]
```diff
@@ -51,6 +53,12 @@ modes: ``compact-process`` (compact mode), ``combined-process`` (combined mode),
       ``styleClass`` (optional): add CSS Classes to HTML DOM of the widget
 
       ``style`` (optional): add inline style to HTML DOM of the widget
+      
+   ``showFullscreenMode``: visibility of the fullscreen mode icon. The default
+   value is ``true``, set to ``false`` to hide the icon
+   
+   ``showWidgetInfo``: visibility of the widget information icon. The default
+   value is ``true``, set to ``false`` to hide the icon
```
Comment (lndanh-axonivy) [inline Documentation/portal-guide/source/portal-developer-guide/dashboard/new-dashboard-process-widget.rst:61]: Add dot (.) at the end of the sentence.

File: Documentation/portal-guide/source/portal-developer-guide/dashboard/new-dashboard-process-widget.rst [hunk 2]
```diff
@@ -51,6 +53,12 @@ modes: ``compact-process`` (compact mode), ``combined-process`` (combined mode),
       ``styleClass`` (optional): add CSS Classes to HTML DOM of the widget
 
       ``style`` (optional): add inline style to HTML DOM of the widget
+      
+   ``showFullscreenMode``: visibility of the fullscreen mode icon. The default
+   value is ``true``, set to ``false`` to hide the icon
```
Comment (lndanh-axonivy) [inline Documentation/portal-guide/source/portal-developer-guide/dashboard/new-dashboard-process-widget.rst:58]: Add dot (.) at the end of the sentence.

File: AxonIvyPortal/portal/src_hd/ch/ivy/addon/portal/generic/dashboard/component/StatisticChartWidgetConfiguration/StatisticChartWidgetConfiguration.xhtml [hunk 3]
```diff
@@ -70,6 +70,23 @@
           <p:messages id="statistic-widget-validation-messages" closable="true" styleClass="ui-g-12"/>
         </div>
       </h:panelGroup>
+      
+      <h:panelGroup id="fullscreen-mode-group" rendered="true"
```
Comment (lndanh-axonivy) [inline AxonIvyPortal/portal/src_hd/ch/ivy/addon/portal/generic/dashboard/component/StatisticChartWidgetConfiguration/StatisticChartWidgetConfiguration.xhtml:74]: Is it alway "render=true"?

File: AxonIvyPortal/portal/src/ch/ivy/addon/portalkit/enums/DashboardWidgetType.java [hunk 4]
```diff
@@ -23,10 +23,11 @@ public boolean canEnableQuickSearch() {
...
 
   public boolean canShowWidgetInfoOption() {
-    return this == TASK || this == CASE;
+    return this == TASK || this == CASE || this == PROCESS;
   }
 
   public boolean canShowFullscreenModeOption() {
-    return this == TASK || this == CASE;
+    return this == TASK || this == CASE || this == PROCESS || this == NEWS || this == CUSTOM || this == PROCESS_VIEWER
+        || this == STATISTIC;
```
Comment (lndanh-axonivy) [inline AxonIvyPortal/portal/src/ch/ivy/addon/portalkit/enums/DashboardWidgetType.java:31]: Is is widget type WELCOME available for this mode?

Comment (lmluat-axonivy) [reply AxonIvyPortal/portal/src/ch/ivy/addon/portalkit/enums/DashboardWidgetType.java:31]: As default welcome widget does not have expand button

**Review by lndanh-axonivy — APPROVED** [review]
LGTM

**Pull request merged**